pages/_document.tsx

```diff
@@ -0,0 +1,18 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+class CustomDocument extends Document {
+  render() {
+    return (
+      <Html>
+        <Head />
+        <body>
+          <Main />
+          <NextScript />
+          <script async src="https://apis.google.com/js/api.js" />
```

Comment:
**gapi** comes from this **API** @AnandBaburajan . So it's not declared anywhere.